akka-docs/src/main/paradox/typed/cluster-singleton.md

```diff
@@ -114,15 +114,15 @@ See @ref:[Downing](cluster.md#downing).
 
 ## Example
 
-Any `Behavior` can be run as a singleton. E.g. a basic counter:
+Any @apidoc[Behavior](typed.Behavior) can be run as a singleton. E.g. a basic counter:
 
 Scala
 :  @@snip [SingletonCompileOnlySpec.scala](/akka-cluster-typed/src/test/scala/docs/akka/cluster/typed/SingletonCompileOnlySpec.scala) { #counter }
 
 Java
 :  @@snip [SingletonCompileOnlyTest.java](/akka-cluster-typed/src/test/java/jdocs/akka/cluster/typed/SingletonCompileOnlyTest.java) { #counter }
 
-Then on every node in the cluster, or every node with a given role, use the `ClusterSingleton` extension
+Then on every node in the cluster, or every node with a given role, use the @apidoc[ClusterSingleton](typed.ClusterSingleton) extension
```

Comment:
```suggestion
Then on every node in the cluster, or every node with a given role, use the @apidoc[ClusterSingleton$](typed.ClusterSingleton) extension
```

Reply:
@raboof  Thank you. I'll use `@apidoc[ClusterSingleton$]` and remove `(typed.ClusterSingleton)`